doc/innovation_roadmap.rst

```diff
@@ -0,0 +1,218 @@
+       Includes an option to calculate dynamic storage capacities for thermal energy storage. Also includes aquifer thermal energy storage, and supplemental heating such as booster heat pumps.
+   * - Modelling of E-fuels 
+           - Allow transportation of e-methanol, e-methane and e-kerosene via pipelines, ships or tanker trucks
+           - Incorporate full load hours into spatial optimisation of e-fuel refineries and supply infrastructure
+     - <TODO>
+   * - Higher granularity topology
+           - Adding more nodes per country and differentiating between prosumer and non-prosumer households for accurate grid interaction modeling
+     - PyPSA-Eur supports multiple resource classes for wind and solar per region to improve accuracy at low spatial resolutions. It also allows for behind-the-meter rooftop PV modeling.
+   * - Improved modelling of prosumer demand
+     - Open TYNDP allows connection of microgeneration e.g. residential solar PV to be connected to low voltage buses.
+       Residential and utility scale PV are treated separately, with separate rules in the workflow to build and cluster rooftop potentials.
+       Open TYNDP also distinguishes between stationary battery storage and EV batteries.
+   * - Consider peaking units as expansion candidates
+     - PyPSA-Eur is a capacity expansion model by nature and can be set to build new peaking units whenever they are the cost-optimal way to ensure reliability.
+   * - Check on remaining CO2 emissions in 2050
+     - TODO
+   * - Implementation of hybrid electrolyser plants
+           - Implementing plants connected to both dedicated renewables and the grid to optimize production and market coupling
+     - Open-TYNDP implements offshore wind hubs where wind farms can connect to both the network and P2G units for H2 production. PyPSA-Eur added minimum unit dispatch settings for electrolysis.
+   * - Hydrogen imports and pipeline assessment
+           - Modeling practical volumes and prices for pipeline imports to evaluate energy security and dependence.
+     - PyPSA-Eur implemented renewable energy imports for H2, ammonia, methanol, and oil with configurable prices and volume limits.
+   * - Geographical correlation in hydrogen production
+     - Open-TYNDP ensures geographical correlation by attaching planning-year dependent renewable profiles from the PECD to specific generators within interconnected zones
+
+Stakeholder Reference Group (SRG) Proposals
+===========================================
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1    
+
+   * - Stakeholder Reference Group
+     - Open TYNDP Innovation Roadmap
+   * - Synthetic Fuels
+           - Incorporating methanol for the maritime sector to align with decarbonization goals and identify infrastructure needs.
+     - PyPSA-Eur introduced methanol-based technologies (e.g., biomass-to-methanol) in its 2024.09 release. Open-TYNDP defaults maritime demand to methanol.
```

Comment:
Methanol can be used also in various other sectors (e.g. as back up power, in industry, as kerosene). See publication here https://arxiv.org/html/2505.09277v1